games-action/minetest/metadata.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<herd>games</herd>
```

Comment:
The primary maintainer is the games herd. I was the previous maintainer (and importer) and I worked on this ebuild with the help of the games herd.

I am well (more then you think) aware of the problems with the games herd (I am not one of them). But just removing them from metadata.xml files is not a clean solution. Decisions about maintainership are not up to proxy-maintainers.